### ISSUE TITLE
AMO (Long Term Solution)

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-it/ctms-api#262]
* AMO generate specific use case what is needed from User stand point
* We set it up

First item: We need to meet with the team to understand their user stories and what they need.

- - -
Condensed summary:

**Issue:** AMO - Doesn't show who is opted out. Acoustic - Does not show which contacts are active developers. 

**The Need:** Sync a relational database for select add-on data to Acoustic data schema. Add on contact info and add-on data needed to create recipient lists for sending mass emails through Acoustic. 

**Currently:** Manual process dependent on 4 people and 1 business day (including Brandon and Tina) They send 5 mass emails to a developer audience per quarter. 33k subscribers.

┆Issue is synchronized with this [Jira Story](https://mozilla-hub.atlassian.net/browse/CTMS-4)

**Comment by data-sync-user** [comment]
➤ Sarah Chu commented:

Target 4/30 to be completed

**Comment by data-sync-user** [comment]
➤ Lauren Niolet commented:

Summary of what we need to do here.

https://docs.google.com/document/d/1CpojSp-YxDx7rccS__bwCK922ytEfOCRkPkFpNyeoMI/edit# ( https://docs.google.com/document/d/1CpojSp-YxDx7rccS__bwCK922ytEfOCRkPkFpNyeoMI/edit# )

TL;DR: Create a relational table in Acoustic that is sync'd from CTMS. This table includes this information from the AMO system ( https://docs.google.com/spreadsheets/d/1WMsA9ukyLVlyiNtveVijdsbfdcDMkkhsGZ56NrGjMQQ/edit#gid=0 ). 1x per day is plenty of frequency.

**Comment by data-sync-user** [comment]
➤ Lauren Niolet commented:

And to be clear, this work should be prioritized AFTER the June 1 migration deadline. None of the data AMO wants sync'd is currently in SFDC so there is no need to do this work before we lose sfdc.

**Comment by leplatrem** [comment]
Acoustic is gone since #896
